### DISCUSSION TITLE
2.7.6 Update. MacOS (Intel) not supported?

### DISCUSSION THREAD

**Discussion by heviiguy** [keepassxreboot/keepassxc#9757] [Q&A] [answered]
I see that the newest update is available for Apples with the Silicon processors. I don't see anything for those of us running older machines. Is anything in the works so that we will also be able to upgrade?

**Comment by phoerious** [comment, marked as answer]
Click on "See more options".

**Reply by heviiguy** [reply]
Oops. I missed that. I went to the Release page where I didn't see the applicable option.

Thanks, Phoerious.
